test/quantization/core/test_quantized_op.py

```diff
@@ -2778,6 +2778,36 @@ def test_qlinear_legacy(self, batch_size, input_channels, output_channels):
         self.assertEqual(Y_fp32, Y_fp32_ref,
                          msg="torch.ops.quantized.fbgemm_linear_dynamic results are off")
 
+    @skipIfNoFBGEMM
+    @given(
```

Comment:
should we use loops instead of hypoothsis here?

Reply:
removed it :)